crypto/rsa/rsa_sign.c

```diff
@@ -13,63 +13,185 @@
 #include <openssl/rsa.h>
 #include <openssl/objects.h>
 #include <openssl/x509.h>
+#include <openssl/md2.h>
```

Comment:
These have only been included to get the *_DIGEST_LENGTH definitions.

Reply:
Maybe add a comment?